test/api/application/create_test.go

```diff
@@ -0,0 +1,70 @@
+package application
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func TestApplicationCreate(t *testing.T) {
```

Comment:
Looks like every test function has an TestApplication prefix that is redundant in the `application` package.  Everything in the package is implied to be scoped to application.
Recommend:
For example: just `TestCreate()` instead of: `TestApplicationCreate()`.  (less stuttering please).